s3/variables.tf

```diff
@@ -34,18 +34,11 @@ variable "replication"{
   default = false
 }
 variable "user_tags" {
-  description = "Mandatory user tags"
-  type = object({
-    team    = string
-    purpose = string
-    owner   = string
-  })
+  description = "User tags"
+  type = map(string)
```

Comment:
= should be in same position as the = in previous line

Reply:
done